src/docs/src/cluster/tls_erlang_distribution.rst

```diff
@@ -36,10 +36,11 @@ the ``certificate`` and its ``private key``.
     .. code-block:: text
 
         [{server,
-          [{certfile, "</path/to/erlserver.pem>"},
+          [{certfile, "<absolute_path/to/erlserver.pem>"},
```

Comment:
removing the leading `/` makes it a relative path, even if the name is clearer.

Reply:
Add leading `/`, thanks for your review.